src/components/Dashboard/cells/time.js

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+
+class Time extends React.Component{
+  constructor(props){
+    super(props);
+    this.state = this.getTime();
+    
+    this.interval = null;
+
+    this.setTime = this.setTime.bind(this);
+
+  }
+
+  getTime(){
+    const today = new Date();
+    var hours = today.getHours();
+    var minutes = today.getMinutes();
+
+    const isAm = hours < 12; 
+
+    if (hours === 0) {
+      hours = 12;
+    }
+    if (!isAm) {
+      hours -= 12;
+    }
+
+    if (hours < 10) {
+      hours = this.addLeadingZero(hours);
```

Comment:
Maybe write a function called `pad` which adds a zero if the string is too short? Then you don't need to write a new if statement every time you call it